emperor/support_files/js/color-view-controller.js

```diff
@@ -56,62 +55,108 @@ define([
     var SLICK_WIDTH = 25, scope = this;
     var name, value, colorItem;
 
+    // Create scale div and checkbox for whether using scalar data or not
+    this.$scaleDiv = $('<div></div>')
```

Comment:
Can you describe these properties in the documentation above? Also maybe some of these need to be private properties, I'll leave that to you, let me know if you would like to talk through that.